clap_derive/src/derives/args.rs

```diff
@@ -1,33 +1,94 @@
-// Copyright 2018 Guillaume Pinot (@TeXitoi) <texitoi@texitoi.eu>,
-// Kevin Knapp (@kbknapp) <kbknapp@gmail.com>, and
-// Andrew Hobden (@hoverbear) <andrew@hoverbear.org>
-//
-// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
-// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
-// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
-// option. This file may not be copied, modified, or distributed
-// except according to those terms.
-//
-// This work was derived from Structopt (https://github.com/TeXitoi/structopt)
-// commit#ea76fa1b1b273e65e3b0b1046643715b49bec51f which is licensed under the
-// MIT/Apache 2.0 license.
-use proc_macro2::TokenStream;
-use proc_macro_error::abort;
-use quote::{quote, quote_spanned};
-use syn::{punctuated::Punctuated, spanned::Spanned, token::Comma, Field, Ident, Type};
```

Comment:
Tangential, but is there any rustfmt option that groups the `crate` and `dep` and `std` imports and always formats them in a certain order? Currently we have different orders in different files and my other projects also have varied orders.

Reply:
https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=#group_imports

Not stable but we could always configure it one off and do a cleanup commit.